### PR TITLE
{debugger,devel,math}[GCCcore/12.2.0] GDB v13.2, ISL v0.26, makeinfo v7.0.3

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
taurusml5 - Linux RHEL 7.6, POWER, 8335-GTX, 6 x NVIDIA Tesla V100-SXM2-32GB, 440.64.00, Python 2.7.5
See https://gist.github.com/Flamefire/dcd9aff6332f8d05a5dba668277b6d87 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/6ffe2b10eb16ac25592357be08eef6ad for a full test report.

File: easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+configopts = '--with-system-zlib --with-system-readline --with-expat=$EBROOTEXPAT '
+configopts += '--with-python=$EBROOTPYTHON/bin/python --with-isl=$EBROOTISL --with-mpc=$EBROOTMPC '
+configopts += '--enable-tui --enable-plugins --disable-install-libbfd '
+
+sanity_check_paths = {
+    'files': ['bin/gdb', 'bin/gdbserver'],
+    'dirs': [],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-12.2.0.eb:43]: Can we add some simple `sanity_check_commands` here?
```suggestion

sanity_check_commands = [
    'gdb --help',
    'gdbserver --help',
]

```

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-12.2.0.eb (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-12.2.0.eb:43]: That is new, but why not. Makes sense I guess.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
taurusml4 - Linux RHEL 7.6, POWER, 8335-GTX, 6 x NVIDIA Tesla V100-SXM2-32GB, 440.64.00, Python 2.7.5
See https://gist.github.com/Flamefire/33f8ac3d80f1246bb1c70a48e790d4f8 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @Flamefire!

**Pull request merged**